components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/IdentityRecoveryConstants.java

```diff
@@ -384,6 +403,7 @@ public String toString() {
                 "SelfRegistration.VerificationCode.SMSOTP.ExpiryTime";
 
         public static final String ENABLE_EMIL_VERIFICATION = "EmailVerification.Enable";
+        public static final String ENABLE_EMAIL_VERIFICATION_ON_UPDATE = "EmailVerificationOnUpdate.Enable";
```

Comment:
Possibly this feature will refer to configs under 'EmailVerification' right.
So it would be bit confusing if another root level is to be mapped